packedForest/src/forestTypes/basicForests/rfClassification/rfTree.h

```diff
@@ -103,7 +111,10 @@ namespace fp{
 
 				inline void checkOOB(){
 					totalOOB += nodeQueue.back().returnOutSampleSize();
-					correctOOB += nodeQueue.back().returnOutSampleCorrect(tree.back().returnClass());
+					//correctOOB += nodeQueue.back().returnOutSampleCorrect(tree.back().returnClass());
+					for(auto &j : nodeQueue.back().returnOutSampleVec()){
+						indexAndVote.push_back(std::vector<int>{j, tree.back().returnClass()});
```

Comment:
also confusing here -- please add commment

looks like you get the outsamples in leaf node, then iterate over them, adding to indexandvote the class at a particular column of a matrix, but i'm not sure how you get the correct row

Reply:
I'm storing the id and vote as a pair, so the index into the matrix doesn't matter.
// Get oob_indices and vote from leaf node as
// <(oob_id, vote)> and push back onto the
// indexAndVote object.

Reply:
don't you need the index into the matrix for the prediction at the forest level?

Reply:
I'm storing the (index, vote) so the index into the matrix does not correspond to the observation id.